src/__init__.py

```diff
@@ -51,7 +51,8 @@ def initConfig():
     config_root.user = ConfigText(fixed_size=False)
     config_root.password = ConfigPassword()
     config_root.sitemap = ConfigText(default="default", fixed_size=False)
-    config_root.refresh = ConfigNumber(3)
+    config_root.refresh = ConfigSelection(default=5, choices=[1, 2, 3, 5, 10, 20])
```

Comment:
With such set o values it is impossible to choose 3 seconds which the default previously. Also seem my other comments regarding refresh time.

Reply:
Done